internal/cmp/resource_network.go

```diff
@@ -138,12 +142,24 @@ func (r *resNetwork) Delete(ctx context.Context, d *utils.Data, meta interface{}
 	return err
 }
 
-func alignNetworkReq(request *models.CreateNetwork) {
-	request.Site.ID = request.GroupID
-	if request.NetworkDomainID != 0 {
-		request.NetworkDomain = &models.IDModel{ID: request.NetworkDomainID}
```

Comment:
why did you remove "if request.NetworkDomainID != 0 {
		request.NetworkDomain = &models.IDModel{ID: request.NetworkDomainID}"

Reply:
updated